kalibro_client/processor/repository.py

```diff
@@ -114,12 +114,12 @@ def has_ready_processing(self):
     def has_processing_after(self, date):
         return self.request(":id/has_processing/after",
                             params={'id': self.id,
-                                    'date': date})['has_processing_in_time']
+                                    'date': str(date)})['has_processing_in_time']
```

Comment:
See my [other comment](https://github.com/mezuro/kalibro_client_py/pull/36/files#r36321779) about the dates. We either should use `isoformat()` here, or better, treat all dates in the `request` method, since they can show up inside nested objects and such.